vmdb/spec/models/vmdb_database_setting_spec.rb

```diff
@@ -22,6 +22,34 @@
     expect(setting.vmdb_database).to eql(@db)
   end
 
+  it 'aliases min_val to minimum_value' do
+    setting = VmdbDatabaseSetting.where('min_val is not null').first
+    expect(setting.min_val).to eql(setting.minimum_value)
```

Comment:
Unless you really *meant* a `.eql?` comparison (vs `==` which is what `.eq` does)